vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
Very good point. AFAICT this is just imitating what was done before the patch, by means of the interwined Hook system. @gares may comment more, but I think this overload of exceptions is not necessary anymore and should go away in further cleanups. However, we need to sort out other parts first.